source/grid/grid_generator.inst.in

```diff
@@ -105,6 +115,11 @@ namespace GridGenerator \{
       const bool                       );
   
     template void
```

Comment:
I get a warning with gcc 4.7 <code>comparison of unsigned expression < 0 is always false</code> for <code>dim = 1</code> from line 452 <code>for(unsigned int i=0; i<dim-1; ++i)</code> in grid_generator.cc, maybe add <code>#if deal_II_dimension != 1</code> here.